app/views/topics/show.html.erb

```diff
@@ -8,6 +8,19 @@
     <%= link_to 'Edit Topic', edit_topic_path(@topic.id) %>
   <% end %>
 
+  </br>
+
+  <% if user_signed_in? %>
+    <h3> Add to collection </h3>
+
+    <%= form_with model: @collections_topic, url: {controller: "collections_topics", action: "create"} do |f| %>
+     <%= select_tag 'add_topic_to_collection', options_from_collection_for_select(@collection_list, 'id', 'name') %>
```

Comment:
This should either be a multi-select, or checkboxes. The associations allow a topic to be assigned to multiple collections.

Reply:
Yup! Will do :)